modules/planning/math/piecewise_jerk/path_time_qp_problem.cc

```diff
@@ -30,6 +30,11 @@ void PathTimeQpProblem::SetRefX(std::vector<double> x_ref) {
   x_ref_ = std::move(x_ref);
 }
 
+void PathTimeQpProblem::SetDirivativePenalty(std::vector<double> penalty_dx) {
+  CHECK_EQ(penalty_dx.size(), num_of_knots_);
```

Comment:
let's not add new CHECK and replace it with proper error handling. CHECK can only be added during initialization stage.